com.unity.render-pipelines.high-definition/Runtime/Lighting/LightLoop/GlobalLightLoopSettings.cs

```diff
@@ -27,6 +27,18 @@ public enum CubeReflectionResolution
         CubeReflectionResolution4096 = 4096
     }
 
+    /// <summary>
+    /// Available graphic formats for the cube reflection probes.
+    /// </summary>
+    [System.Serializable]
+    public enum ReflectionProbeFormat
```

Comment:
ReflectionProbeFormat
We need to say ReflectionAndPlanarProbeFormat as it apply to both, should be explicit in the enum and the code